src/phoenix/server/api/types/DimensionShape.py

```diff
@@ -0,0 +1,21 @@
+from enum import Enum
+
+import strawberry
+
+from phoenix.core.model_schema import CONTINUOUS, Dimension
+
+
+@strawberry.enum
+class DimensionShape(Enum):
+    continuous = "continuous"
+    discrete = "discrete"
+
+    @classmethod
+    def from_(cls, dim: Dimension) -> "DimensionShape":
```

Comment:
Did you mean `from_dimension` here?

Reply:
I'm with you, that feels more traditional. But I think I was trying to emulate @RogerHYang 's syntax here https://github.com/Arize-ai/phoenix/blob/c6297389fcfab48d1afdf5b8fb37faac295cd96a/src/phoenix/server/api/types/DimensionType.py#L24

My guess is that roger wants it to read like a sentence `from_(dimension)` - I'm happy to refactor in this PR if we agree that explicit method names are better (I honestly thought this was a typo when I saw it.

Reply:
yes, "brevity is the soul of wit" haha. i have no problem if it's refactored

Reply:
You sound like you would love Haskell ;)